packages/core-mobile/app/new/common/components/GlobalLoadingState.tsx

```diff
@@ -0,0 +1,19 @@
+import { ActivityIndicator, View } from '@avalabs/k2-alpine'
+import React from 'react'
+
+export const GlobalLoadingState = (): React.JSX.Element => {
+  return (
+    <View
+      sx={{
+        justifyContent: 'center',
+        alignItems: 'center',
+        position: 'absolute',
+        top: 0,
+        left: 0,
+        right: 0,
+        bottom: 0
```

Comment:
I think this should use relative position as it will be used in a component and not a modal/full screen